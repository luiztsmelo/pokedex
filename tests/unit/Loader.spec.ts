import { shallowMount } from '@vue/test-utils';
import Loader from '@/components/Loader.vue';

describe('Loader.vue', () => {
  test('renders correctly', () => {
    const wrapper = shallowMount(Loader)
    expect(wrapper.element).toMatchSnapshot()
  })
});
