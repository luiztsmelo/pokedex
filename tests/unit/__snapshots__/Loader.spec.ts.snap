// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader.vue renders correctly 1`] = `
<div
  class="loader"
/>
`;
